src/cljs/athens/events.cljs

```diff
@@ -1884,23 +1884,23 @@
 
 (reg-event-fx
   :unlinked-references/link
-  (fn [_ [_ {:block/keys [string uid] :as block} title]]
+  (fn [_ [_ {:block/keys [string uid]} title]]
     (js/console.debug ":unlinked-references/link:" uid)
-    (if-let [local? (not (client/open?))]
+    (if client/open?
       (let [unlinked-references-link-event (common-events/build-unlinked-references-link -1 uid string title)
             tx-data                        (resolver/resolve-event-to-tx @db/dsdb unlinked-references-link-event)]
-        (js/console.debug ":unlinked-references/link: local?" local?)
+        (js/console.debug ":unlinked-references/link:" tx-data)
         {:fx [[:dispatch [:transact tx-data]]]})
-      {:fx [[:dispatch [:remote/unlinked-references-link block title]]]})))
+      {:fx [[:dispatch [:remote/unlinked-references-link string uid title]]]})))
 
 
 (reg-event-fx
   :unlinked-references/link-all
   (fn [_ [_ unlinked-refs title]]
     (js/console.debug ":unlinked-references/link:" title)
-    (if-let [local? (not (client/open?))]
+    (if client/open?
```

Comment:
Same thing, remote will be executed when not connected and local when connected.

Reply:
Updated